packages/react-vapor/src/components/dropdownSearch/MultiSelectDropdownSearch/DraggableSelectedOption.tsx

```diff
@@ -93,7 +95,7 @@ export class DraggableSelectedOption extends React.PureComponent<IDraggableSelec
                 <div className="selected-option-wrapper" style={{opacity}}>
                     <SelectedOption
                         {..._.omit(this.props, DraggableSelectedOptionPropsToOmit)}
-                        label={this.props.isDragging ? null : this.props.label}
+                        label={this.props.isDragging ? null : this.props.selectedTooltip?.title ?? this.props.label}

```

Comment:
Considering `props.selectedTooltip` is now of type `ITooltipProps`, I think we should modify our implementation of `SelectedOption` component a bit, so that it respects what we set in the `selectedTooltip` prop.

For example, if my item has the following props:
```ts
selectedTooltip: {
    title: 'whatever',
    placement: TooltipPlacement.Bottom,
}
```
I would expect the tooltip of the selected value to appear on the bottom, while with the current implementation it will appear on top.